README.md

```diff
@@ -15,3 +15,50 @@ the function and run automatically when you commit through a GitHub action.
 ## Runtime Analysis
 
 What is the worst-case big $\Theta$ time complexity of your algorithm?
+
+## Answer
+
+For my algorithm the worst-case Big $\Theta$ time complexity is $\Theta(V!\cdot V^{2})$.
+
+We can determine this by analysing the different parts of this algorithm's implementation.
+
+- `are_isomorphic()`: Checks that both graphs have the same number of vertices and edges. Then starts the process of checking for isomorphism.
+  - Vertex and edge comparison have a constant time complexity.
+  - Calls `createAdjMatrix()` to convert graph1 to an adjacency matrix.
+  - Then calls `checkPermutations()` To check all permutations of graph2.
+
+- `checkPermutations()`: This function generates all permutations of the vertices in graph2 and compares the adjacency matrix of each permutation.
+  - Calls `createAdjMatrix()`.
+  - Calls `areMatricesEqual()`.
+  - Iterates through all permutations which is $V!$.
```

Comment:
Then your overall complexity can't be correct.

Reply:
Could you explain a little more to why this is not correct? I thought it made sense, since later  in the answer we analysis the time complexity for both those functions and get the cost of each permutation.

Reply:
Ah sorry, I missed the first part in your complexity expression. This is fine.

Reply:
Ok, Thank you!